src/viewer/styles.styl

```diff
@@ -108,6 +108,13 @@
             max-width   500px
             height      auto
 
+    .pho-viewer-canceled
+        @extend $empty
+        &:before
+            background  embedurl('./assets/icon-cloud-wrong.svg') center center / contain no-repeat
+        h2
+            color white
```

Comment:
I guess you could add a top margin to the button to lighten the view a little bit.
For example :
```
button
    margin-top 1rem
```

Reply:
Je pense que le fait que le bouton soit trop large est aussi un souci, comment ferais-tu ?

Reply:
`flex-grow 0` sur le bouton ne change rien.

Reply:
Je sais pas trop faut que je plonge dedans et que je vois ce qu'il y a à l'heure actuelle. :/
Une autre carte peut être ?